internal/services/securitycenter/security_center_contact_resource.go

```diff
@@ -42,6 +36,13 @@ func resourceSecurityCenterContact() *pluginsdk.Resource {
 		},
 
 		Schema: map[string]*pluginsdk.Schema{
+			"name": {
```

Comment:
since we are adding this property we'll need to update the docs?

Reply:
Absolutely, i have not yet run the scaffold website command.

Reply:
the docs still need updating & we'll need to make this Optional/Defaulted in 3.x / make this Required in 4.0 since otherwise this is a breaking change